db/oracle_schema.rb

```diff
@@ -464,11 +464,11 @@
   add_index "countries", ["code"], name: "index_countries_on_code"
 
   create_table "deleted_objects", force: :cascade do |t|
-    t.integer  "owner_id",    limit: 19, precision: 19
+    t.integer  "owner_id",    precision: 38
```

Comment:
Oops! This may be related to [THREESCALE-2298 - Fix Oracle indices for api_docs_services referencing services.id](https://issues.jboss.org/browse/THREESCALE-2298)